model/src/main/java/mops/antworten/TextAntwort.java

```diff
@@ -1,10 +1,12 @@
 package mops.antworten;
 
+import java.util.Random;
 import lombok.AllArgsConstructor;
-import lombok.Value;
+import lombok.Data;
 
-@Value
+@Data
```

Comment:
Please don't use Data here. Just manually add the Getter Setter and EqualsAndHashCode Annotations. So the ones you actually need.